common/defaults.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2016 The go-ethereum Authors
```

Comment:
This isn't the right place for these type of settings. `common` isn't your average dump place for settings ;-)